### DISCUSSION TITLE
2024/09/27 社区周会纪要

### DISCUSSION THREAD

**Discussion by yuluo-yx** [alibaba/spring-ai-alibaba#34] [General]
# 2024/09/26 社区周会

## 基础能力建设

1. 基于 open api 适配更多 LLM 能力，多模态，语音模型 等
2. 调研相关业内框架，完善 Spring AI Alibaba 框架能力
3. 对话记忆，RAG rerank 等基础能力

### Example 完善与优化

扩展 Spring AI Alibaba 的 Example，提供更多的 Example 使用场景。

1. 本地调用模型示例
2. sql 查询示例
3. 文本分类示例，给一些词语，按词性，分成积极，消极，中性
4. 完善与优化 playground 和 部分 stream 流式 API 调用

## 可观测

Spring AI Alibaba 可观测建设：建设 Spring AI Alibaba 应用的可观测能力。

https://docs.spring.io/spring-ai/reference/observabilty/index.html

结合可观测领域组件，调研 AI 框架的可观测性，例如 OpenTelemetry  对 AI 可观测标准的讨论等

## 开发工具

结合 Spring Al Alibaba 构建开发工具 

开发工具： https://github.com/firebase/genkit ；

运维工具：结合 AI 对日志进行分析，并给出优化建议和事故原因等。

## Multi-Agent 

https://microsoft.github.io/autogen/
https://github.com/crewAIInc/crewAI/
https://github.com/OpenBMB/ChatDev/

**Comment by wangdadatao** [comment]
希望加入工作流的编排能力，这样更灵活

**Comment by YouHa-UHA** [comment]
什么时候可以加入对graph相关能力的处理呢？
